todo_rxbloc_app/lib/feature_manage_todo/views/manage_todo.dart

```diff
@@ -0,0 +1,145 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_rx_bloc/flutter_rx_bloc.dart';
+import 'package:provider/provider.dart';
+import 'package:todos_repository_core/todos_repository_core.dart';
+
+import '../../app_extensions.dart';
+import '../../base/enums/new_todo_enum.dart';
+import '../bloc/manage_todo_bloc.dart';
+
+class ManageTodoPage extends StatefulWidget {
+  const ManageTodoPage({super.key, this.todo});
+  final TodoEntity? todo;
+
+  @override
+  State<ManageTodoPage> createState() => _ManageTodoPageState();
+}
+
+class _ManageTodoPageState extends State<ManageTodoPage> {
+  @override
+  void initState() {
+    noteController.text = widget.todo?.note ?? '';
+    taskController.text = widget.todo?.task ?? '';
+    super.initState();
+  }
+
+  final TextEditingController taskController = TextEditingController();
+  final TextEditingController noteController = TextEditingController();
+  @override
+  Widget build(BuildContext context) {
+    return GestureDetector(
+      onTap: () => FocusScope.of(context).unfocus(),
+      child: Column(
+        children: [
+          _someToastMessage(context),
+          Expanded(
+            child: Scaffold(
+              appBar: AppBar(
+                title: Text(context.l10n.featureAddTodo.addTodoAppBarTitle),
+              ),
+              body: RxBlocBuilder<ManageTodoBlocType, bool>(
+                state: (bloc) => bloc.states.isLoading,
+                builder: (context, snapshot, bloc) => snapshot.hasData
+                    ? Padding(
+                        padding: EdgeInsets.all(
+                            context.designSystem.spacing.space16),
+                        child: Column(
+                          children: [
+                            RxBlocBuilder<ManageTodoBlocType, String>(
```

Comment:
Please use the `RxTextFormFieldBuilder` Look in the article below for more information

https://medium.com/prime-holding-jsc/building-forms-in-flutter-454b8d65f48